index.src.html

```diff
@@ -519,14 +520,18 @@ <h3 id="fetch-integration">Fetch Integration</h3>
   execute the following step:
```

Comment:
Nit: Can you change the line above this to refer to "step #12" while you're here?